R/run-model.R

```diff
@@ -99,6 +99,8 @@ hintr_run_model <- function(data, options, output_path = tempfile(),
   progress$complete("Generating uncertainty ranges")
   progress$start("Preparing outputs")
   progress$print()
+  ## TODO: Include input data in output package based on model options
```

Comment:
does this interact with #30?

Reply:
I think a separate thing - Jeff wanted a flag users could tick to download all the input data along with their outputs. I'm expecting @jeffeaton to add some code which will include the input data in that download based on this new flag. 

Perhaps I should just add a ticket for that and do it to save Jeff some time which he can spend on the model instead.